src/xtgeo/interfaces/rms/_rms_base.py

```diff
@@ -0,0 +1,102 @@
+"""Shared base for RMS RegularSurface reader/writer."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from xtgeo.common.log import null_logger
+from xtgeo.interfaces.rms._rmsapi_package import rmsapi
+from xtgeo.interfaces.rms.rmsapi_utils import (
+    RmsApiUtils,
+    _StorageTypeRegularSurface as StorageType,
+)
+
+if TYPE_CHECKING:
+    from xtgeo.interfaces.rms._rmsapi_package import (
+        RmsProjectOrPathType,
+        RmsProjectType,
+    )
+
+logger = null_logger(__name__)
+
+__all__ = ["_BaseRmsDataRW"]
```

Comment:
```suggestion
```

It's a bit unusual to put `__all__` in a module, particularly a private one. Is there a use case?